package.json

```diff
@@ -46,8 +46,7 @@
     "expect": "^1.14.0",
     "json": "^9.0.4",
     "mocha": "^3.2.0",
-    "mversion": "^1.10.1",
-    "react": "^15.0.1"
```

Comment:
why remove react? It's needed in tests

Reply:
It's already listed in `dependencies`. Being in `devDependencies` is redundant and causes a collision warning from Yarn.

Reply:
Ok, maybe we should also remove it from dependencies, ultimately this is a dependency at the user level not ours.

Reply:
what do you think?

Reply:
Actually, since it's _only_ needed to run the tests it should be in `devDependencies` and not in `dependencies`. 
I'll switch what I did to that.

Reply:
Now I'm second guessing myself. If we remove React from `dependencies` then the requirement for it will fall through to `react-element-to-jsx-string`'s `peerDependencies` right?

```
"peerDependencies": {
  "react": "^0.14.8 || ^15.0.1 || ^16.0.0 || ^16.0.0-rc || ^16.0.0-beta || ^16.0.0-alpha"
},
```
 Just wondering if this could cause breakage for anyone? Maybe a change like that shouldn't go along with this PR.

Reply:
I don't know either true, I would say react should be a peer dep of expect-jsx too
What happens when:
- moda
  - peerdep react
  - dep modb
  - modb
    - peerdep react

I don't know but that would be a good finding because ultimately that's the state we should be in. Maybe let it as it was for now and submit another PR if you are up to it? Thanks